index.html

```diff
@@ -16,5 +16,166 @@
 <body>
     <!-- load javascript after loading all html content -->
     <script src="script/script.js"></script>
+    <br> </br>
+    <br> </br>
+    <h1> This is Jayan's Website</h1>
+    <br> </br>  <!-- This creates a space between the two lines of text -->
```

Comment:
1. What is your intent with this code?  What are you hoping to make the website look like?

2. I would avoid using `<br>` except when formatting raw text. `<br>` tends to behave differently on different devices and has weird behaviors. I would rely on `display: flex`, `margin` and `padding` to do what you want.

3. You only need the `<br>` not a `</br>`:

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br